packages/@react-spectrum/view/src/SVGIllustration.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {classNames, filterDOMProps, useSlotProps, useStyleProps} from '@react-spectrum/utils';
+import {DOMProps, SVGProps, ViewStyleProps} from '@react-types/shared';
+import {HTMLElement} from 'react-dom';
+import React, {ReactElement, ReactSVGElement, RefObject} from 'react';
+
+export interface IllustrationProps extends DOMProps, SVGProps, ViewStyleProps {
+  children: ReactElement | ReactElement[]
+}
+
+export const SVGIllustration = React.forwardRef((props: IllustrationProps, ref: RefObject<HTMLElement>) => {
+  props = useSlotProps(props, 'illustration');
+  console.log('props', props);
+  let {
+    children,
+    ...otherProps
+  } = props;
+  let {styleProps} = useStyleProps(otherProps);
+
+  return (
+    <svg {...filterDOMProps(otherProps)} {...styleProps} className={classNames({}, styleProps.className)} ref={ref}>
```

Comment:
So I don't think this is what we'd want for a component like SVGIllustration. People would expect to pass something like this to it
<svg><path ...></svg>
this is because when an svg is exported from illustrator or other programs, it comes with the svg tag already.
for now, lets avoid this situation and not use this component. Instead of including a illustration, lets just target it directly with css. I'll add a comment up top in the css file explaining what to do there

Reply:
I didn't target the svg with css because we pass aria props to the illustration. https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/%40react-aria/illustrated-message/src/useIllustratedMessage.ts#L59-L62 Wouldn't I have to find the SVG child, clone it and add these props if this isn't it's own component?

Reply:
Ah, yes, it does need to be a component then that takes slots. Good!